lib/cocoapods/target.rb

```diff
@@ -305,6 +311,14 @@ def mark_application_extension_api_only
       @application_extension_api_only = true
     end
 
+    # compiles the target with Swift's library evolution support,
```

Comment:
this does not look right in terms of formatting. Capitalize 'C'.

ah I also see above its messed up can you fix that also?